src/storage-adapters/mongo/index.js

```diff
@@ -75,13 +75,14 @@ export default class MongoAdapter {
       return {};
     }
 
-    const { _id, content, mimeType, fetchDate } = record;
+    const { _id, content, mimeType, fetchDate, isRefilter } = record;
 
     return {
       id: _id.toString(),
       content,
       mimeType,
       fetchDate: new Date(fetchDate),
+      isRefilter: !!isRefilter,
```

Comment:
Why do you need to cast it to boolean here ?

Reply:
To avoid returning `undefined` when the property does not exist. I think when a variable is named `isSomething` or `hasSomething`, it should always be a boolean.

Reply:
I'm not such a fan of double-bangs 🙂
Details: https://medium.com/@MattiSG/when-should-i-use-a-double-bang-1672bf8e677f
```suggestion
      isRefilter: Boolean(isRefilter),
```

Reply:
I think `!!` is not so cryptic and is very common, so I'm not really sensible to the argument of the related article but if you have a personal preference of using `Boolean()` I'm ok with that :)

Reply:
Also not finding it cryptic but fine with `Boolean` :-)